expfmt/text_parse.go

```diff
@@ -73,9 +73,9 @@ type TextParser struct {
 	// These tell us if the currently processed line ends on '_count' or
 	// '_sum' respectively and belong to a summary/histogram, representing the sample
 	// count and sum of that summary/histogram.
-	currentIsSummaryCount, currentIsSummarySum     bool
-	currentIsHistogramCount, currentIsHistogramSum bool
-	currentMetricIsInsideBraces                    bool
+	currentIsSummaryCount, currentIsSummarySum                      bool
+	currentIsHistogramCount, currentIsHistogramSum                  bool
+	currentMetricIsInsideBraces, currentMetricInsideBracesIsPresent bool
```

Comment:
Can we get a comment explaining what these mean?

Reply:
Addressed in https://github.com/prometheus/common/pull/691/commits/acd0c007ef78f0864cf33cfdd8d0acb7b8a2b61a, thanks for the feedback